packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.spec.ts

```diff
@@ -135,6 +139,9 @@ describe('resourceCollection', () => {
       type: RumEventType.RESOURCE,
       _dd: {
         discarded: false,
+        resolveDuration: (50 * 1e6) as ServerDuration,
+        durationDiff: 50000000 as ServerDuration,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏for consistency sake
```suggestion
        resolveDuration: (50 * 1e6) as ServerDuration,
        durationDiff: (50 * 1e6) as ServerDuration,
```